luarules/gadgets/game_autocolors.lua

```diff
@@ -626,18 +660,18 @@ else	-- UNSYNCED
 	setUpAllLocalTeamColors()
 
 	local iconDevModeColors = {
-		armblue       = armBlueColor,
-		corred        = corRedColor,
-		scavpurp      = scavPurpColor,
-		raptororange  = raptorOrangeColor,
-		gaiagray      = gaiaGrayColor,
+		armblue = armBlueColor,
+		corred = corRedColor,
+		scavpurp = scavPurpColor,
+		raptororange = raptorOrangeColor,
+		gaiagray = gaiaGrayColor,
 	}
 
 	local iconDevMode = Spring.GetModOptions().teamcolors_icon_dev_mode
 	local iconDevModeColor = iconDevModeColors[iconDevMode]
 
 	local function isDiscoEnabled()
-		return anonymousMode == "disco" and not Spring.GetSpectatingState()
+		return anonymousMode == "disco" and not mySpecState
```

Comment:
As far as I can see, this is one of the only changes that's not just linting. Why would this break team colors?